themes/default/content/docs/intro/concepts/resources/providers.md

```diff
@@ -377,3 +377,24 @@ class MyStack
 {{% /choosable %}}
 
 {{< /chooser >}}
+
+While default providers are enabled by default, they [can be disabled]({{< relref
+"/docs/intro/concepts/config#special-configuration-options" >}}) on a per stack basis. You might want to do this if
+passing an explicit providers is required for all created resources. For example, to disable the `aws` provider, you can
```

Comment:
Perhaps worth expanding slightly on the reasons this may be desirable? "To avoid accidentally depending on the default provider", "to ensure your programs must be explicit about which provider they will use".